geonode/maps/views.py

```diff
@@ -604,8 +607,8 @@ def new_map_config(request):
                     bbox = list(layer_bbox[0:4])
                 else:
                     bbox[0] = min(bbox[0], layer_bbox[0])
-                    bbox[1] = max(bbox[1], layer_bbox[1])
-                    bbox[2] = min(bbox[2], layer_bbox[2])
+                    bbox[1] = min(bbox[1], layer_bbox[1])
+                    bbox[2] = max(bbox[2], layer_bbox[2])
```

Comment:
CC @Gustry 
This is the part which gets funny because it compares layer.bbox which is in [x0,y0,x1,y1] format for the first layer bbox, but comparing it with the others as if the format is [x0,x1,y0,y1]. Then when the user saves the map, it will be saved and swapped.